_includes/footer.html

```diff
@@ -6,7 +6,7 @@
     </div>
     <div>
         <h4>About Us</h4>
-        <a href="/history-partners-and-advisory-board">History, Partners, and Advisory Board</a>
+        <a href="/about">History, Partners, and Advisory Board</a>
```

Comment:
I was skeptical about this but it actually makes sense

Reply:
This is a problem for later, but I think it would be nice if we could automatically generate this list and just use the `title` attribute from the page. That way we don't have to repeat this whole name again and again.